### DISCUSSION TITLE
Multi-Ranger Deck interfacing with ESP32

### DISCUSSION THREAD

**Discussion by pgorenkin21** [bitcraze/discussions#1124] [Q&A]
Can anyone provide guidance or resources on interfacing an ESP32 with a Crazyflie Multiranger? I'm currently working on a project involving these components and am facing challenges in finding relevant documentation or tutorials. Specific tips or links to helpful resources would be greatly appreciated! Thanks!

**Comment by knmcguire** [comment]
Hi! I suggest looking at this [blogpost](https://www.bitcraze.io/2022/12/victors-thesis-leave/), where @victorhook explained work for his thesis with an updated type of multi-ranger. He used an ESP module for that.
